src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/main.tf

```
@@ -279,6 +279,32 @@ module "jupyterhub-openid-client" {
     "developer" = ["jupyterhub_developer", "dask_gateway_developer"]
     "analyst"   = ["jupyterhub_developer"]
   }
+  client_roles = [
+    {
+      "name" : "allow-app-sharing-role",
+      "description" : "Allow app sharing for apps created via JupyterHub App Launcher (jhub-apps)",
+      groups : [],
```

It's odd that linter did not harmonize this

```suggestion
      "groups" : [],
```